src/components/InfoBlock.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import { RadioButton } from './RadioButton';
+import { SORTINGS } from '../data/data';
+
+import './info-block.scss';
+
+export function InfoBlock() {
+    return (
+        <div className="info-block">
+            <span className="movies-count">0 movies found</span>
+            <div className="sorting-block">
+                <p>Sort by</p>
+                {SORTINGS.map(item =>
```

Comment:
Code style
```
{
  SORTINGS.map(item =>
    <RadioButton
      key={item.id}
      item={item}
      isChecked={item.checked}
    />)
}
```

FYI: It should be enough to pass only item and inside component get props.